lib/internal/errors.js

```diff
@@ -98,7 +98,6 @@ function sysErrorMessage(prefix, ctx) {
 // and may have .path and .dest.
 class SystemError extends Error {
   constructor(key, context = {}) {
-    context = context || {};
```

Comment:
FWIW, This is not really a duplicated default. This is there to catch the possibility of something like `new SystemError('foo', null)`, which is not handled by the default argument.

Reply:
The current way `SystemError`s are instantiated in the code base warrants that `context` would at least be a `{}`, theoretically we don't even need to handle the default argument if we are being careful with new `SystemError`s.